### ISSUE TITLE
Release 0.0.6 fails to load after upgrade

### ISSUE THREAD

**Issue by bj00rn** [custom-components/nordpool#148]
## Version of the custom_component

0.0.6

## Homeassistant version

2022.3.2

## Configuration

```yaml

Add your logs here.

```

## Describe the bug
Integtation fails to load aftet upgrade to 0.0.6

## Debug log

Logger: homeassistant.setup
Source: setup.py:162
First occurred: 19:26:23 (1 occurrences)
Last logged: 19:26:23

Setup failed for custom integration nordpool: Unable to import component: Exception importing custom_components.nordpool

Felet härrör från en anpassad integration.

Logger: homeassistant.loader
Source: custom_components/nordpool/events.py:31
Integration: nordpool (documentation, issues)
First occurred: 19:26:23 (1 occurrences)
Last logged: 19:26:23

Unexpected exception importing component custom_components.nordpool
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 553, in get_component
    cache[self.domain] = importlib.import_module(self.pkg_path)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/nordpool/__init__.py", line 17, in <module>
    from .events import async_track_time_change_in_tz
  File "/config/custom_components/nordpool/events.py", line 31, in <module>
    action: Callable[[datetime], Awaitable[None] | None],
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'NoneType'

```text

Add your logs here.

```

**Comment by nvanggaard** [comment]
I can confirm this - i'm having the exact same issue, updated from 0.5 to 0.6 and get the same error as Bj00rn

**Comment by dimkin-eu** [comment]
same here 0.0.6 + Home Assistant Core 2022.5.3 + Home Assistant Supervisor 2022.05.1

```
Logger: homeassistant.loader
Source: custom_components/nordpool/events.py:31
Integration: nordpool ([documentation](https://github.com/custom-components/nordpool/), [issues](https://github.com/custom-components/nordpool/issues))
First occurred: 21:27:08 (4 occurrences)
Last logged: 21:33:11

Unexpected exception importing platform custom_components.nordpool.sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 603, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 620, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/nordpool/__init__.py", line 17, in <module>
    from .events import async_track_time_change_in_tz
  File "/config/custom_components/nordpool/events.py", line 31, in <module>
    action: Callable[[datetime], Awaitable[None] | None],
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'NoneType'
```

sensor config
```
- platform: nordpool
  VAT: True
  currency: "EUR"
  region: "LV"
  precision: 4
  low_price_cutoff: 0.95
  price_type: kWh
  friendly_name: "nordpool_price"
```

**Comment by nvanggaard** [comment]
The master branch is working though.

**Comment by Hellowlol** [comment]
What Python version are you using?

 Anyway though I had released 0.0.7. Seems like I just bumped it.

**Comment by dimkin-eu** [comment]
python 3.9. 
looks like it could be connected with hacs ( v. 1.24.5)
--


<br class="Apple-interchange-newline">

```
Logger: custom_components.hacs
Source: custom_components/hacs/tasks/base.py:34
Integration: HACS ([documentation](https://hacs.xyz/docs/configuration/start), [issues](https://github.com/hacs/integration/issues))
First occurred: 21:24:05 (1 occurrences)
Last logged: 21:24:05

<HacsTask prosess_queue> failed: Repository structure for master is not compliant
```

**Comment by Hellowlol** [comment]
The problem is your python version was to old to support the new typing

**Comment by eyvindj** [comment]
0.0.7 just showed up and solved my problems, they were identical before since upgrading core and nordpool from 0.0.5 to 0.0.6

**Comment by Hellowlol** [comment]
Like magic 😁 anyway, my bad as I mentioned, I thought I had released 0.0.7 but I forgot to do it.
